[2024-04-25T21:15:22.143+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-25T21:15:22.191+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.task_7 manual__2024-04-25T21:14:42.515218+00:00 [queued]>
[2024-04-25T21:15:22.205+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.task_7 manual__2024-04-25T21:14:42.515218+00:00 [queued]>
[2024-04-25T21:15:22.206+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2024-04-25T21:15:22.233+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): task_7> on 2024-04-25 21:14:42.515218+00:00
[2024-04-25T21:15:22.245+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'task_7', 'manual__2024-04-25T21:14:42.515218+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/python_op.py', '--cfg-path', '/tmp/tmpxecgxdly']
[2024-04-25T21:15:22.249+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1922) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-25T21:15:22.250+0000] {standard_task_runner.py:91} INFO - Job 154: Subtask task_7
[2024-04-25T21:15:22.251+0000] {standard_task_runner.py:63} INFO - Started process 1923 to run task
[2024-04-25T21:15:22.333+0000] {task_command.py:426} INFO - Running <TaskInstance: my_dag.task_7 manual__2024-04-25T21:14:42.515218+00:00 [running]> on host f9a12f9b65d4
[2024-04-25T21:15:22.532+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='task_7' AIRFLOW_CTX_EXECUTION_DATE='2024-04-25T21:14:42.515218+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-25T21:14:42.515218+00:00'
[2024-04-25T21:15:22.533+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-25T21:15:22.554+0000] {logging_mixin.py:188} INFO - 7
[2024-04-25T21:15:27.555+0000] {logging_mixin.py:188} INFO - test
[2024-04-25T21:15:27.557+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-25T21:15:27.559+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-25T21:15:27.576+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=task_7, execution_date=20240425T211442, start_date=20240425T211522, end_date=20240425T211527
[2024-04-25T21:15:27.623+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-25T21:15:27.655+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-25T21:15:27.660+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
